MAINTAINERS.yml

```diff
@@ -4712,6 +4712,14 @@ West:
   labels:
     - "platform: TI"
 
+"West project: hal_wch":
```

Comment:
probably needs a `labels` entry too, and the corresponding GH label to be created too?

Reply:
not necessary, the manifest action should add `manifest-hal_wch` automatically, I would not bother with an extra one, I'll create the manifest one once the repo is approved

Reply:
actually not even, it's already there, guess the bot auto-created it

Reply:
deleted the other one